app/classes/textile.rb

```diff
@@ -33,17 +35,36 @@ class Textile < String
 
   URL_TRUNCATION_LENGTH = 60 unless defined?(URI_TRUNCATION_LENGTH)
 
+  ########## Class methods #####################################################
+
   # Convenience wrapper on instance method Textile#textilize_without_paragraph.
   def self.textilize_without_paragraph(str, do_object_links = false,
                                        sanitize = true)
     new(str).textilize_without_paragraph(do_object_links, sanitize)
   end
 
+  # Wrap self.textilize_without_paragraph, marking output trusted safe
+  def self.textilize_without_paragraph_safe(str, do_object_links = false,
```

Comment:
Does this method, and `textilize` safe belong here, or in StringExtensions?

Reply:
Because it's about textile, this seems like the right place for it.